ghcategorize.py

```diff
@@ -186,9 +186,10 @@ def checkForBotCommand(json, commandList):
         return None
     for command in commandList:
         # FIXME: bot may check for commands in the middle of a comment?
-        if json['body_text'].startswith(command):
-            user, date = getUserDate(json)
-            return user, date
+        if not json['body_text'] is None:
```

Comment:
This seems like a reasonable addition esp. based on your sympy experience (I love sympy).

For readability/maintainability, would you mind moving `not` so it reads:
```
if json['body_text'] is not None:
```